docs/learn/learn-parachains-protocol.md

```diff
@@ -0,0 +1,107 @@
+---
+id: learn-parachains-protocol
+title: Parachains' Protocol Overview
+sidebar_label: Protocol Overview
+description: High level view of actors and protocols involved in running parachains on Polkadot.
+keywords: [parachains, application-specific, sharding, protocol, validator, collator, nominator]
+slug: ../learn-parachains-protocol
+---
+
+:::info
+
+This page is a summary of the [Protocol Overview chapter in **The Polkadot Parachain Host Implementer's Guide**](https://paritytech.github.io/polkadot/book/protocol-overview.html)
+
+:::
+
+The aim of the protocol is to carry a parachain's block from authoring to inclusion through a process that can be carried out repeatedly and in parallel for each parachain connected to the relay chain.
+
+## Main Actors
+
+### Validators
+
+They are responsible for validating proposed parachain's blocks by checking the **Proof-of-Validity** (PoV) of the blocks and ensuring the PoV remains available. They have "skin in the game", meaning they have funds bonded on-chain that can be partially or fully confiscated in case of misbehavior.
+
+### Collators
+
+They create the PoV that validators know how to check. Creating PoV requires familiarity with transaction format and block authoring rules of a specific parachain, as well as having access to its full state.
+
+### Fishermen
+
+They catch misbehaving validators in exchange for a bounty. Collators and validators can behave as fishermen.
+
+## Summary View
+
+The parachains' protocol can be summarized into three main steps:
+
+1. Collators send to validators parachain blocks with their PoV for check.
+2. Validators validate the block using the PoV, signing statements that can have positive or negative.
+3. With enough positive statements the block can be added to the relay chain.
+
+The figure below shows a representation of a parachain with collators and validators. The figure also shows the journey of a block through the three main steps, as well as the sections where the [Inclusion Pipeline](#inclusion-pipeline) and the [Approval Process](#approval-process) take place (see next Sections).
+
+![parachain-protocol-summary](../assets/parachain-protocol-summary.png)
```

Comment:
Some questions if I put myself in the shoes of a general Wiki reader

- Why is a parablock shown as a block with ?
- What is the difference between white and yellow tick marks (if they are positive and negative, then green and red colors would be a better choice).
- Can a validator sign multiple statements on the parablock or a single statement?

The gray shade can be a bit lighter.

Reply:
right, let me address this on Monday, I will review the figures and change the text accordingly

Reply:
- parablock with ? and white tick is a block that has been produced by a collator. The ? means that the block is not valid yet and the white tick is the PoV by the collator.

- the white tick is the PoV from the collator, the yellow tick are PoV from the para-validators

I can add a legend to explain the different symbols.

- as far as I know a validator can sign a single statement

Note that the network block contains multiple parablocks, I need to change this in the diagram.

I will provide an alternative diagram with lighter gray shade.

Let me know if this will answer your points :)

Reply:
Makes sense. Add this info as the figure's description.

Reply:
Alright, the information has been added in the text with the new commits.